vendor/k8s.io/kubernetes/pkg/proxy/iptables/proxier_test.go

```diff
@@ -411,7 +411,7 @@ func NewFakeProxier(ipt utiliptables.Interface) *Proxier {
 		nodePortAddresses:        make([]string, 0),
 		networkInterfacer:        utilproxytest.NewFakeNetwork(),
 	}
-	p.syncRunner = async.NewBoundedFrequencyRunner("test-sync-runner", p.syncProxyRules, 0, time.Minute, 1)
+	p.syncRunner = async.NewBoundedFrequencyRunner("test-sync-runner", p.SyncProxyRules, 0, time.Minute, 1)
```

Comment:
Please squash this commit ("call the openshift wrapper for syncProxyRules") into "UPSTREAM: <drop>: preserve old health/metrics behavior", and change it to use `forceSyncProxyRules` rather than `SyncProxyRules`

Reply:
done